modules/virt-vmware-comparison.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/about_virt/about-virt.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="virt-vmware-comparison_{context}"]
+= Comparison of {VirtProductName} with VMware vSphere
+
+{VirtProductName} is conceptually different from VMware vSphere, much of its functionality comes from the underlying {product-title}, and it therefore does not offer direct alternatives to individual vSphere concepts and components. That said, if you are familiar with VMware vSphere, the following table lists the most common vSphere concepts and outlines what {VirtProductName} components you can use to accomplish a similar task.
+
+.Mapping of vSphere concepts to their closest {VirtProductName} counterparts
+[options="header"]
+[cols="2,2,3"]
+|===
+|vSphere concept |{VirtProductName} |Explanation
+
+|Datastore
+|Persistent volume (PV){nbsp}+ +
+Persistent volume claim (PVC)
+|Stores VM disks. A persistent volume represents existing storage and is attached to a VM through a persistent volume claim. When created with the `ReadWriteMany` (RWX) access mode, persistent volume claims can be mounted by multiple VMs simultaneously.
+
+|Dynamic Resource Scheduling (DRS)
+|Pod eviction policy{nbsp}+ +
+Descheduler
+|Provides active resource balancing. A combination of pod eviction policies and a descheduler allows VMs to be live migrated to more appropriate nodes to keep node resource utilization manageable.
```

Comment:
Suggestion to take or leave:
```suggestion
|Provides active resource balancing by allowing VMs to be live migrated to more appropriate nodes. This can keep node resource utilization manageable.
```